README.md

```diff
@@ -104,6 +114,32 @@ Host your own Chocolatey package repository
 
 #### Parameters
 
+##### `chocolatey_server_app_pool_name`
+Set apppool name used by the chocolatey.server website. Defaults to
+'chocolatey.server'.
+
+##### `compatibility_mode`
+Compatibility mode was added to allow this module to be backwards
+compatible with older implementations. Defaults to 'true'.
+
+##### `disable_default_website`
+The default website is stopped to prevent conflicts with the
+chocolatey server website. An alternate port can be defined and
+this option can be set to disabled so it doesn't modify the default
+website. Defaults to 'true'.
+
+##### `default_website_path`
+The default website path is needed in order to modify the default
+website. Defaults to 'C:\inetpub\wwwroot'.
+
+##### `packages_folder`
+An alternate folder can be defined for the .nupkg files.
+
+##### `packages_folder_permissions`
+Set permissions on packages folder. Defaults to:
+  * IIS APPOOL\${chocolatey_server_app_pool_name} - modify
+  * IIS_IUSRS - modify
```

Comment:
Interesting, how is this actually passed? Would prefer an example of this one in the readme.